astropy/coordinates/tests/test_name_resolve.py

```diff
@@ -155,7 +155,7 @@ def test_name_resolve_cache(tmpdir):
 
         urls = get_cached_urls()
         assert len(urls) == 1
-        assert 'http://cdsweb.u-strasbg.fr/cgi-bin/nph-sesame/' in urls[0]
+        assert any([urls[0].startswith(x) for x in sesame_url._value])
```

Comment:
I think this is equivalent and doesn't use private attributes

```suggestion
        assert any([urls[0].startswith(x) for x in sesame_url.get()])
```

Also, cool thing I recently learned: using generators instead of lists means ``any`` can halt after the 1st success. Marginally faster, but not worth the trouble here. Just thought I'd share.

E.g.
```
def will_fail(i):
    if i == 5:
        raise ValueError
    else:
        return i
    
any(j == 3 for j in (will_fail(i) for i in range(6)))  # succeeds
```

Reply:
@nstarman , good to know, thanks! I agree in this case, given the max length is 2, that would be premature optimization. Still, good to know for other work. 😸

Reply:
Also, good call on `get()`. I will use that!